stdlib/base.pkl

```diff
@@ -1890,9 +1964,18 @@ class Mapping<out Key, out Value> extends Object {
 
   /// The keys contained in this mapping.
   external keys: Set<Key>
+  
+  /// The values contained in this mapping.
+  external values: List<Value>
+  
+  /// The entries contained in this mapping.
+  external entries: List<Pair<Key, Value>>
```

Comment:
Is there a particular reason why there are `List`s while `keys` is a `Set`? Mappings are ordered (and documented as such), so this sort of makes sense to me. Having this reasoning documented might be helpful.

Reply:
IMHO these should be `Listing`s, and `keys` should be changed to `Listing` via a deprecation cycle.